pkg/genericapiserver/genericapiserver.go

```diff
@@ -583,10 +583,24 @@ func verifyEtcdServersList(options *options.ServerRunOptions) {
 	}
 }
 
+func verifySecurePort(options *options.ServerRunOptions) {
+	if options.SecurePort < 1 || options.SecurePort > 65535 {
+		glog.Fatalf("--secure-port %v must be between 1 and 65535, inclusive", options.SecurePort)
+	}
+}
+
+func verifyInsecurePort(options *options.ServerRunOptions) {
+	if options.InsecurePort < 1 || options.InsecurePort > 65535 {
```

Comment:
Thanks for reminding. I check this file and find it seems that 0 is the switch for SecurePort (L658, L678). I will check it again when I go back to office. If so, I think we should modify the condition of verifySecurePort and it's message.